<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeCategoryName</class>
 <widget class="QDialog" name="ChangeCategoryName">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QListWidget" name="conductorCategoriesLW"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="okPB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QLineEdit" name="newNameLE"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="cancelPB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>New Name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="applyPB">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>conductorCategoriesLW</tabstop>
  <tabstop>newNameLE</tabstop>
  <tabstop>okPB</tabstop>
  <tabstop>cancelPB</tabstop>
  <tabstop>applyPB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
